tide-log/src/lib.rs

```diff
@@ -0,0 +1,65 @@
+#![feature(async_await)]
+#![deny(
+    nonstandard_style,
+    rust_2018_idioms,
+    future_incompatible,
+    missing_debug_implementations
+)]
+
+use futures::future::BoxFuture;
+use log::{info, trace};
+use tide::{
+    middleware::{Middleware, Next},
+    Context, Response,
+};
+
+macro_rules! box_async {
```

Comment:
Following the merging of #245, the `box_async` macro should now be avoided, instead bringing `FutureExt` into scope instead and using it that way. As said here also https://github.com/rustasync/tide/pull/222#discussion_r285783972

Given it's a relatively recent change, but I think with this change we can finish merging it!

Reply:
Would someone like to pick this up at the moment? (my branch is editable to maintainers)
 (I'm a bit of a rush today, but this and #241 are otherwise ready to merge, I think)

Reply:
I can wrap it up for you real quick, sure!

Reply:
@fairingrey awesome! thank you :smiley: